ColorCode_prv.h

```diff
@@ -0,0 +1,26 @@
+#ifndef COLORCODE_PRV_H
+#define COLORCODE_PRV_H
```

Comment:
what does _PRV stands for ?

Reply:
It represents a private header. The API's in this file won't be available publically